.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo 'deb [trusted=yes] https://repo.goreleaser.com/apt/ /' | tee /etc/apt/sources.list.d/goreleaser.list
```

Comment:
Is this needed? you install goreleaser in line 15

Reply:
No it is not :)

Seems like I forgot about this when I worked on the devcontainer. Seems like I also forgot about a `/` that also should be removed.